docs/sources/administration/data-source-management/teamlbac/_index.md

```diff
@@ -20,7 +20,19 @@ Label-Based Access Control (LBAC) allows fine-grained access control to data sou
 
 ## Supported Data Sources
 
+### Feature availability
+
 LBAC for data sources is currently generally available for `Loki` and in **experimental** for `Prometheus`. Support for additional data sources may be added in future updates.
+The following table shows the current availability of data source permissions for different data sources:
```

Comment:
I would remove the previous sentence if you're adding a table - they're conveying the same info, but I think I prefer the table format.